front/app/containers/MainHeader/UserMenu/index.tsx

```diff
@@ -56,7 +58,10 @@ const UserMenu = () => {
           className="intercom-user-menu-button"
           id="e2e-user-menu-dropdown-button"
         >
-          <User userId={userId} />
+          <User
+            userId={userId}
+            showVerificationBadge={isMobileOrSmaller ? false : true}
+          />
```

Comment:
Chatted with Rob, and the value of showing the words "Verified" or "Unverified" to residents is low. Here we remove it from the mobile view to give the rest of the elements more space on the mobile nav bar.